Security/src/AuthClient/Program.cs

```diff
@@ -0,0 +1,69 @@
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpOverrides;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Steeltoe.Common.Certificate;
+using Steeltoe.Configuration.CloudFoundry;
+using Steeltoe.Configuration.CloudFoundry.ServiceBinding;
+using Steeltoe.Management.Endpoint;
+using Steeltoe.Security.Authentication.OpenIdConnect;
+using Steeltoe.Security.Authorization.Certificate;
+using System;
+using Steeltoe.Samples.AuthClient;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Configuration
+    .AddCloudFoundry() // needed for actuators
+    .AddCloudFoundryServiceBindings()
+    .AddAppInstanceIdentityCertificate(new Guid("a8fef16f-94c0-49e3-aa0b-ced7c3da6229"), new Guid("122b942a-d7b9-4839-b26e-836654b9785f"));
```

Comment:
These GUIDs look magical, I propose to extract them in constants (in both projects) to clarify what they represent.